### DISCUSSION TITLE
Yeni mezun olan biri için ideal şehir

### DISCUSSION THREAD

**Discussion by ademilter** [ademilter/homepage#46] [Q&A] [answered]
Geliştirici sohbetleri kanalındaki videonuzu izleyerek mail atıyorum. Lise son sınıf öğrencisiyim ve üniversite sınavlarına hazırlanıyorum. Bilgisayar mühendisi olacağım ama nerede olmak istiyorum diye kendime sorunca sadece İstanbul geliyor. İstanbul dışında başka bir yer istemiyorum. Çünkü bilgisayar dünyasının kalbi İstanbul'da atıyor gibi. Konferanslar, seminerler, iş olanakları olsun her şey orada sayılır. O yüzden aklımda sadece İstanbul var. Bana bu konuda önerebileceğiniz bir şey var mıdır ? 

Teşekkür ederim :)

**Comment by okandavut** [comment]
İzmir'deyim çok kararsız kalıyorum gelip gelmeme konusunda yanıtını bekleyenlerden biri de benim güzel soru :)

**Comment by ademilter** [comment, marked as answer]
Zaten başka şehir düşünseydin yanlış yapardın :) 
Yeni başlayan biri için istanbul olmazsa olmaz (türkiye sınırları içinde). Senin de bahsettiğin gibi bu işlerin kalbi malesef istanbul.

Benim videodaki rahatsızlığım biraz bunalmışlık sendromu. Uzun yıllar sonra gelen bir durum. Yaşın ilerlemesi ve piyasaya doymuşluk hissi gelince artık kalabalık şehirde yaşamak istemiyorsun ki zaten istanbul gibi bir metropolün hızı ve gürültüsü adama fazla geliyor.

Benim hızlı gelişmeme çok büyük katkı sağlamıştır bu şehir. Her gün yeni etkinlikler, yeni insanlar, işten çıkıp aynı gün başka bir işe girmeler, freelance piyasası, çalışacağın kafeler ve sayamadığım bir sürü imkan ve kolaylık.

Yeni mezun birirnin pişmesi ve gelişmesi için bütün imkanlar mevcut. Geriye kalan tek şey iyi bir firma. Burada iyiden kasıt, fazla maaş, iyi patron, eğlenceli ofis değil. Bir firmayı iyi yapan orada çalışan insnalardır. Bilgisine ve tecrübesine güvendiğin insnaların yanında çalışman kendine vereceğin en büyük hediye olacaktır. İlk zamanlar kazandığın parayı çok önemsemeyebilirsin. Kazanacağın tecrübe ve birikim zamanla o yılların hakkını fazlasıyla ödeyecektir.

Bitirmeden;
İstanbul dışında da iyi insanları bir araya getirip güzel işler yapılacağını gösteren @atolye15 gibi firmaların çoğalması dileğiyle.

**Comment by okandavut** [comment]
Soruyu ben sormadım ancak aklımdaki bir soruydu. Güzel bir cevap. Atolye15 için çabalıyoruz bakalım 👍  Teşekkürler @ademilter

**Comment by lahmacun** [comment]
Yapılan en büyük hatalardan birisi de bu, herkes hemen zengin olma, iyi para kazanma peşinde, kimse ne bildiğini ve tecrübesini zerre kadar umursamıyor. Kendi reklam ajansımızı kurmak gibi bir girişimimiz oldu, iki senedir ordaki bütün web işleriyle ilgileniyorum ve ciddi anlamda az para kazanıyorum ama o kadar çok tecrübe ediniyorum ki orda. Yeni başlayan arkadaşlara tavsiyemdir, gidin ücretsiz bile olsa bir yerde çalışmaya başlayın, ne kadar geliştiğinize inanamayacaksınız.

**Comment by borantula** [comment]
İstanbul'un pek çok durum için neredeyse alternatifsiz ilk seçenek olması maalesef doğru. Maalesef diyorum çünkü kişisel olarak ilk başlarda imkanları nedeniyle çok cazip gelse bile, Adem'in de dediği gibi, uzun vadede çok yıpratıyor; ülke bazında ise tüm yumurtalar aynı sepete koyma durumu çok sağlıklı bir denge yaratmıyor.

Ancak son yıllarda çok örneğini gördüğüm bir çalışma şekli de İstanbul'da çalışarak gerekli tecrübe ve networkü edinip sonrasında yaşaması akıl sağlığı açısından daha faydalı ve daha keyifli şehirlerde yaşayıp uzaktan çalışmak. Ben son 3 yıldır yurtdışındayım ancak eğer yurtdışı planlarımız yolunda gitmemiş olsaydı B planım 1 sene içerisinde bu şekilde bir yaşama-çalışma düzeni ayarlamaktı.
